### PR TITLE
Javadoc

### PR DESCRIPTION
**Pull request by jburel**
fix javadoc warnings
In order to see if some warnings remain, you will need to use Java version > 1.8 (tested with Java 11)
Currently no warnings are returned if Java 1.8 is used
To check run gradle javadoc

14 warnings will remain. They should all happen under ``src/main/java/omero/gateway``. This is fixed in the new repo See https://github.com/ome/omero-java-gateway/pull/1

### CODE REVIEW

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [OMERO-build-push#32](https://web-proxy.openmicroscopy.org/west-ci/job/OMERO-build-push/32/). See the [console output](https://web-proxy.openmicroscopy.org/west-ci/job/OMERO-build-push/32/consoleText) for more details.
Possible conflicts:
  - PR #26 mtbc 'adjust some Blitz API responses to signal "OK"'
    - src/main/slice/omero/cmd/Graphs.ice
    - src/main/slice/omero/cmd/Admin.ice
    - src/main/slice/omero/Repositories.ice

--conflicts

File: src/main/slice/omero/Repositories.ice [hunk 1]
```diff
@@ -129,14 +130,15 @@ module omero {
...
-             * Similar to {@link #list} but recursive and returns only
+             * Similar to {@code list} but recursive and returns only
              * primitive values for the file at each location. Guaranteed for
              * each path is only the values id and mimetype.
              *
              * After a call to unwrap, the returned {@link omero.RMap} for a
              * call to treeList("/user_1/dir0") might look something like:
              *
              * <pre>
+             * {@code
```
Comment (mtbc) [inline src/main/slice/omero/Repositories.ice]: These seem to lose the indent. It complains with just the `<pre>` I guess?

Comment (jburel) [reply src/main/slice/omero/Repositories.ice]: yes

File: src/main/slice/omero/api/IAdmin.ice [hunk 2]
```diff
@@ -139,6 +139,7 @@ module omero {
                  * Looks up {@link omero.model.Experimenter} experimenters who
                  * use LDAP authentication  (has set dn on password table).
                  *
+                 * @param id if of the Experimenter. Not null.
```
Comment (mtbc) [inline src/main/slice/omero/api/IAdmin.ice]: "if of" looks wrong

File: src/main/slice/omero/api/RenderingEngine.ice [hunk 3]
```diff
@@ -28,21 +26,21 @@ module omero {
          * planes within the pixels set onto an <i>RGB</i> image.
          *
          * The RenderingEngine allows to fine-tune the settings that
-         * define the transformation context &#151; that is, a specification
+         * define the transformation context that is, a specification
```
Comment (mtbc) [inline src/main/slice/omero/api/RenderingEngine.ice]: maybe i.e. here too would be a clearer break

File: src/main/slice/omero/api/Search.ice [hunk 4]
```diff
@@ -57,38 +57,37 @@ module omero {
...
                  *
                  * @param size maximum number of results per call to
-                 *             {@link #results}
+                 *             {@code results}
                  */
                 idempotent void setBatchSize(int size) throws ServerError;
 
                 /**
-                 * Returns the current batch size. If {@link #setBatchSize}
+                 * Returns the current batch size. If {@code #etBatchSize}
```
Comment (mtbc) [inline src/main/slice/omero/api/Search.ice]: maybe *set*

File: src/main/slice/omero/api/Search.ice [hunk 5]
```diff
@@ -136,14 +135,14 @@ module omero {
...
-                 * overrides the {@link #setUseProjections} setting.
+                 * overrides the {@code setUseProjections} setting.
                  */
                 idempotent void setReturnUnloaded(bool returnUnloaded) throws ServerError;
 
                 /**
                  * Returns the current return-unloaded setting. If true, all
                  * returned entities will be unloaded. If
-                 * {@link #setReturnUnloaded} has not been called, the
+                 * {@code codesetReturnUnloaded} has not been called, the
```
Comment (mtbc) [inline src/main/slice/omero/api/Search.ice]: too many codes :smiley:

Comment (jburel) [reply src/main/slice/omero/api/Search.ice]: never enough!

File: src/main/slice/omero/Repositories.ice (hunk 1 above)
Comment (mtbc) [reply src/main/slice/omero/Repositories.ice]: I'm not getting the complaint if I reverse it (OpenJDK 11.0.2) but I'm also not getting indents so there's no regression from my point of view so happy to let this stand in the absence of being able to compare with `slice2html`.

**Comment by jburel** [comment]
@mtbc any more comments?

**Comment by mtbc** [comment]
No, though it's big enough maybe worth having someone else cast their eye over it also!

**Comment by jburel** [comment]
Merging for m6 
we can adjust when we have slice2html cc @rgozim

**Pull request merged**